R4ML/R/r4ml.frame.R

```diff
@@ -97,21 +97,20 @@ r4ml.calc.num.partitions <- function(object_size) {
 }
 
 
-#' Coerce to a R4ML Frame
+#' Coerce to an R4ML Frame
 #'
-#' Convert a data.frame, r4ml.matrix, or Spark DataFrame into a r4ml.frame
+#' Convert a data.frame, r4ml.matrix, or Spark DataFrame into an r4ml.frame
```

Comment:
In other places I think we use `SparkDataFrame` instead of `Spark DataFrame`. I think we should try to be consistent about this.